lib/class-wp-rest-widgets-controller.php

```diff
@@ -415,10 +415,7 @@ protected function save_widget( $request ) {
 			}
 
 			if ( isset( $request['instance']['raw'] ) ) {
-				if (
-					! isset( $widget_object->show_instance_in_rest ) ||
-					! $widget_object->show_instance_in_rest
-				) {
+				if ( ! $widget_object->show_instance_in_rest ) {
```

Comment:
Why aren't these in the original PR?

Reply:
I did not 100% correctly anticipate what data I'd need on the frontend. I can move the REST API changes over.